tracer/src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -141,8 +141,8 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Initialize(IUnknown* cor_profiler_info_un
 
         if (!functions_worker_runtime_value.empty() && !IsAzureFunctionsEnabled())
         {
-            Logger::Info("DATADOG TRACER DIAGNOSTICS - Profiler disabled: Azure Functions are not officially "
-                         "supported. Enable instrumentation with DD_TRACE_AZURE_FUNCTIONS_ENABLED.");
+            Logger::Info("DATADOG TRACER DIAGNOSTICS - Profiler explicitly disabled for Azure Functions. "
+                         "Enable instrumentation with DD_TRACE_AZURE_FUNCTIONS_ENABLED.");
```

Comment:
Can probably remove the second bit, seeing as they must have set that variable?
```suggestion
            Logger::Info("DATADOG TRACER DIAGNOSTICS - Profiler explicitly disabled for Azure Functions.");
```